templates/managed-ec2-v7.yaml

```diff
@@ -495,7 +440,7 @@ Resources:
     Properties:
       ImageId: !If [HasAMIId, !Ref AMIId, !FindInMap [AWSRegionArch2AMI, !Ref 'AWS::Region', !FindInMap [AWSInstanceType2Arch, !Ref InstanceType, Arch]]]
       InstanceType: !Ref InstanceType
-      KeyName: !Ref KeyName
+      KeyName: !If [HasKeyName, !Ref KeyName, !Ref 'AWS::NoValue']
```

Comment:
May have been asked before, if no value then the instance is only accessible from JumpCloud account?

Reply:
that is correct